keyboards/rotr/kb.h

```diff
@@ -0,0 +1,12 @@
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
+	K00, K01, K02  \
+) { \
+	{ K00,   K01,   K02 }  \
+}
+
+#endif
```

Comment:
This file should be renamed `rotr.h`, also...

```suggestion
#pragma once

#include "quantum.h"

#define LAYOUT( \
	K00, K01, K02  \
) { \
	{ K00,   K01,   K02 }  \
}

```